WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewController.swift

```diff
@@ -244,6 +245,23 @@ private extension DashboardViewController {
         }
     }
 
+    /// Checks if the store has any existing products, to determine whether it is eligible for products onboarding.
+    ///
+    func checkProductsOnboardingEligibility() {
```

Comment:
Maybe `trackedProductsOnboardingEligibility` express the intention better?

```suggestion
    func checkProductsOnboardingEligibility() {
```

Reply:
Makes sense! Updated the name in [3c61ec4](https://github.com/woocommerce/woocommerce-ios/pull/7921/commits/3c61ec4421ab066bd305a16a47694a9602330cc3).